9.5/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -61,6 +61,29 @@ or if it was already present, [`postgres`](http://www.postgresql.org/docs/9.2/st
 is executed and will run as PID 1. You can stop the detached container by running
 `docker stop postgresql_database`.
 
+Supplying custom configuration file
```

Comment:
What about add notice about "how to supply custom configuration file"? This section is mainly about getting configuration template.